internal/internal_workflow.go

```diff
@@ -94,6 +94,7 @@ type (
 	// Note that workflow.Context is used instead of context.Context to avoid use of raw channels.
 	workflow interface {
 		Execute(ctx Context, input []byte) (result []byte, err error)
+		WorkflowType() string
```

Comment:
We represent WorkflowType and ActivityType as a struct on server.  Do you think we should expose similar struct here?

Reply:
I just removed it here. It was a leftover from a previous design attempt.